speech/microphone/transcribe_streaming_mic_test.py

```diff
@@ -19,6 +19,8 @@
 
 from unittest import mock
 
+import google.auth.transport.grpc  # noqa: F401
+import google.auth.transport.requests  # noqa: F401
```

Comment:
Do these need to be imported? We try to avoid noqa wherever possible

Reply:
Yes, without those imports nox can't run all tests. I am also not a big fan of such solutions, if you have a good workaround on your mind, I am all for it :)